parl/remote/scripts.py

```diff
@@ -125,30 +126,50 @@ def start_master(port, cpu_num, monitor_port):
     p = subprocess.Popen(command, stdout=FNULL, stderr=subprocess.STDOUT)
     FNULL.close()
 
-    master_ip = get_ip_address()
-    cluster_info = """
+    monitor_info ="""
```

Comment:
pls use `cluster_info` at line 129 and line 135 as `monitor_info' is being used at line 161.